src/components/cards/CardItem.vue

```diff
@@ -41,9 +41,9 @@
 
 			</transition>
 		</div>
-		<ul class="labels" @click="openCard">
```

Comment:
No need to change the structure, a unordered list is the better fit here I'd say.